.github/workflows/erlang-tests.yml

```
@@ -0,0 +1,146 @@
+        uses: actions/cache@v4
+        with:
+          path: _build
+          key: ${{ runner.os }}-rebar3-${{ matrix.otp }}-${{ hashFiles('rebar.config', 'rebar.lock', 'src/**/*.erl') }}
+          restore-keys: |
+            ${{ runner.os }}-rebar3-${{ matrix.otp }}-
+
+      - name: Make test script executable
+        run: chmod +x ./run_test.sh
+
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)